data/contributor_data.json

```diff
@@ -0,0 +1,126 @@
+{
+    "RAW_DATA": [
+        {
+            "hash": "",
+            "author_msg": "",
+            "author_name": "",
+            "author_email": "",
+            "merge": false,
+            "line_added": 0,
+            "line_removed": 0,
+            "lines_of_code": 0,
+            "complexity": 0,
+            "methods": [],
+            "filename": [],
+            "filepath": []
+        },
```

Comment:
Please keep in mind that this file should be deleted before merging the PR, it was added to master by mistake but it should not be used